reconcile/utils/oc.py

```diff
@@ -971,21 +987,19 @@ def __init__(
         if server:
             self.client = self._get_client(server, token)
             self.api_kind_version = self.get_api_resources()
+            self.api_resources = self.api_kind_version.keys()
```

Comment:
is this used anymore?

Reply:
Yes, used in `cluster_deployment_mapper.py`